lpa-submissions-e2e-tests/cypress/integration/notifyingInterestedParties/notifyingInterestedParties.step.js

```diff
@@ -0,0 +1,58 @@
+import { Given, When, Then, Before } from 'cypress-cucumber-preprocessor/steps';
+import defaultPathId from '../../utils/defaultPathId';
+
+const page = {
+  id: 'interestedPartiesAppeal',
+  heading: 'Notifying interested parties of the appeal',
+  section: 'Optional supporting documents',
+  title: 'Notifying interested parties of the appeal - Appeal Questionnaire - Appeal a householder planning decision - GOV.UK',
+  url: 'notifications',
+}
+
+let disableJs = false;
+
+const goToNotifyingPartiesPage = () => {
+  cy.goToPage(page.url, undefined, disableJs);
```

Comment:
goToPage has two parameters, not three...

Reply:
It USED to have two, now it has 3. AS-10 changed that